src/Microsoft.AspNet.Mvc.Razor/TagHelpers/UrlResolutionTagHelper.cs

```diff
@@ -108,6 +107,11 @@ public override int Order
         /// <inheritdoc />
         public override void Process(TagHelperContext context, TagHelperOutput output)
         {
+            if (output == null)
```

Comment:
Doubt this helper would change _that_ much but I'm not too fussed either way. Will add the extra check.